docs/workflows/verify-images.rst

```diff
@@ -0,0 +1,262 @@
+.. _verify-images:
+
+Configure client signature verification policy
+==============================================
+
+In order to verify container images consumed from Pulp Container Registry the following files need
+to be configured on the client side:
+
+``/etc/containers/registries.d/default.yaml``
+
+This is a default registries.d configuration file.  You may add to this file or create additional
+files in registries.d/.
+
+For full details and spec refer to https://github.com/containers/image/blob/main/docs/containers-registries.d.5.md
+
+Provide read and write location for a separate ``sigstore`` for the registry in question.
+Pulp Container Registry has an integrated signature store, therefore there is no need to provide any
+updates to this file. Client will try to use registry extentions API to read and write image
+signatures.
+
+``/etc/containers/policy.json``
+
+This is a default signature verification policy file that is used to specify policy, e.g. trusted
+keys, applicable when deciding whether to accept an image, or individual signatures of that image,
+as valid.
+
+Parsing for this json file is strict: unrecognized, duplicated or otherwise invalid fields cause the
+entire file, and usually the entire operation, to be rejected.
```

Comment:
```suggestion
Parsing this json file is performed in a strict manner: unrecognized, duplicated or otherwise invalid fields cause the
entire file, and usually the entire operation, to be rejected.
```